Service.cs

```diff
@@ -28,10 +28,7 @@ public OpenVpnService()
             this.AutoLog = true;
 
             _eventLog = new EventLog();
-            if (!EventLog.SourceExists(this.ServiceName))
-            {
-                EventLog.CreateEventSource(this.ServiceName, "Application");
-            }
+            // this assumes that OpenVPNService event source has been created, since it requires elevated privileges

```

Comment:
While this still has issues when there is a pre-existing `OpenVPNService` event source under the `Security` log registry branch, I think we can wait until someone really runs into that particular issue, before we approach fixing it.